src/pipeline/prepare-code-snippet.ts

```diff
@@ -0,0 +1,37 @@
+import { CodeContext } from '../code-formatter';
+import { DIGITAL_AUTO, PYTHON } from '../utils/codeConstants';
+import { PipelineStep } from './pipeline-base';
+
+export class PrepareCodeSnippetStep extends PipelineStep {
+    public execute(context: CodeContext) {
+        context.codeSnippetStringArray = this.removeSubstringFromArray(context.codeSnippetStringArray, DIGITAL_AUTO.VEHICLE_INIT);
+        context.codeSnippetStringArray = this.removeSubstringFromArray(
+            context.codeSnippetStringArray,
+            PYTHON.IMPORT_DEPENDENCY_FROM,
+            PYTHON.IMPORT
+        );
+        context.codeSnippetStringArray = this.removeSubstringFromArray(context.codeSnippetStringArray, PYTHON.COMMENT);
+    }
+
+    private removeSubstringFromArray(array: string[], substringOne: string, substringTwo?: string): string[] {
```

Comment:
If this removes a substring from an array why are there 2 substrings possible as parameters? Should this be named removeSubstring**s**FromArray ?